src/ui/public/state_management/__tests__/state.js

```diff
@@ -13,7 +13,7 @@ import StubBrowserStorage from 'test_utils/stub_browser_storage';
 import EventsProvider from 'ui/events';
 
 describe('State Management', function () {
-  const notify = new Notifier();
+  const notifier = new Notifier();
```

Comment:
I think this works better as a noun as a verb. Also, I think it's a more common convention to see a class instance assume the name of the class but with a lowercase first letter.